config/vufind/ConsoleMenu.yaml

```
@@ -0,0 +1,311 @@
+          arguments:
+            - label: Filename
+              required: true
+          options:
+            - label: Properties File
+              switch: -p
+              type: string
+        - label: Import MARC authority records
+          type: external-command
+          command: import-marc-auth.sh
+          winCommand: import-marc-auth.bat
+          arguments:
+            - label: Filename
+              required: true
+            - label: Properties file
+              required: false
+        - label: Import XML
+          type: internal-command
+          command: import/import-xsl
+        - label: Import CSV
+          type: internal-command
+          command: import/import-csv
+        - label: Import via webcrawl
+          type: internal-command
+          command: import/webcrawl
+    - label: Compile Options
+      type: menu
+      contents:
+        - label: Compile theme (flatten hierarchy)
+          type: internal-command
+          command: compile/theme
+    - label: Completion Options
+      type: menu
+      contents:
+        - label: Dump the shell completion script to enable shell autocompletion
+          type: internal command
+          command: completion
+    - label: Generate Options
+      type: menu
+      contents:
+        - label: Generate a dynamic route
+          type: internal-command
+          command: generate/dynamicroute
+        - label: Generate a subclass, with lookup by class name
+          type: internal-command
+          command: generate/extendclass
+        - label: Generate a service (override with a new child class)
+          type: internal-command
+          command: generate/extendservice
+        - label: Generate a non-tab record action route
+          type: internal-command
+          command: generate/nontabrecordaction
+        - label: Generate a new plugin class
+          type: internal-command
+          command: generate/plugin
+        - label: Generate a record route
+          type: internal-command
+          command: generate/recordroute
+        - label: Generate a static route
+          type: internal-command
+          command: generate/staticroute
+        - label: Generate and configure a new theme
+          type: internal-command
+          command: generate/theme
+        - label: Generate and configure a new theme mixin
+          type: internal-command
+          command: generate/thememixin
+    - label: Harvest Options
+      type: menu
+      contents:
+        - label: Batch import harvested MARC records
+          type: external-command
+          command: harvest/batch-import-marc.sh
+          winCommand: harvest/batch-import-marc.bat
+          arguments:
+            - label: Harvest subdirectory
+              required: true
+            - label: Properties file
+              required: false
+          options:
+            - label: Use the directory path as-is, do not append it to /usr/local/vufind/local/harvest.
```

The file path in this label is dynamically generated, so it might not be true for all installations. Probably better to use the environment variable instead -- it's shorter and more generic:

```suggestion
            - label: Use the directory path as-is, do not append it to $VUFIND_LOCAL_DIR/harvest.
```

---

Fixed!